packages/client/src/runtime/core/request/PrismaPromise.ts

```diff
@@ -3,23 +3,22 @@
  * original `Promise` are optional so that it can be backwards-compatible.
  * @see [[createPrismaPromise]]
  */
-
 export interface PrismaPromise<A> extends Promise<A> {
   /**
    * Extension of the original `.then` function
    * @param onfulfilled same as regular promises
    * @param onrejected same as regular promises
-   * @param transactionId for interactive tx ids
+   * @param txId string for iTX & Promise for regular tx
    */
   then<R1 = A, R2 = never>(
     onfulfilled?: (value: A) => R1 | PromiseLike<R1>,
     onrejected?: (error: unknown) => R2 | PromiseLike<R2>,
-    transactionId?: number,
+    txId?: string,
```

Comment:
This looks like a breaking change in the public type?

Reply:
Actually, it was always typed wrong. That's the tx id for itxs, so it's always a string.